src/pages/home/report/ReportActionsView.js

```diff
@@ -145,6 +145,10 @@ function ReportActionsView(props) {
         });
 
         return () => {
+            if (unsubscribeVisibilityListener.current) {
```

Comment:
Please add it in the useEffect call where we set up unsubscribeVisibilityListener